tests/php/test-amp-twitter-embed-handler.php

```diff
@@ -31,6 +37,8 @@ class AMP_Twitter_Embed_Handler_Test extends WP_UnitTestCase {
 	 */
 	public function setUp() {
 		parent::setUp();
+		$this->prevent_block_pre_render();
```

Comment:
This calls `parent::setUp()` twice, once here, and once in `WithoutBlockPreRendering ::setUp` (which is mapped to `$this->prevent_block_pre_render();` here).